src/contract.rs

```diff
@@ -50,29 +57,11 @@ pub fn execute(
     // Ensure that the message is valid before processing the request
     msg.validate()?;
     match msg {
-        ExecuteMsg::RegisterPayable {
-            payable_type,
-            payable_uuid,
-            scope_id,
-            payable_denom,
-            payable_total,
-        } => register_payable(
-            deps,
-            info,
-            RegisterPayableV1 {
-                payable_type,
-                payable_uuid,
-                scope_id,
-                payable_denom,
-                payable_total,
-            },
-        ),
-        ExecuteMsg::OracleApproval { payable_uuid } => {
-            oracle_approval(deps, info, OracleApprovalV1 { payable_uuid })
-        }
-        ExecuteMsg::MakePayment { payable_uuid } => {
-            make_payment(deps, info, MakePaymentV1 { payable_uuid })
+        ExecuteMsg::RegisterPayable { .. } => {
```

Comment:
While I do like how clean this keeps the `contract.rs` file, I'm not sure it there's a major point in it.  Totally just a stylistic choice, moving these maps to their own function, but repeating the code twice like this might actually be a performance hit?  Idk, but tests still run in like sub-1-ms time lol